packages/thirdweb/src/chainsaw/types.d.ts

```diff
@@ -0,0 +1,98 @@
+import type { Transaction as ViemTransaction } from "viem";
+import type { Hex } from "../utils/encoding/hex.ts";
+import type { NFT as ParsedNFT } from "../utils/nft/parseNft.js";
+
+export type ChainsawResponse<T = unknown> = {
+  data?: T;
+  error?: string;
+  page?: number;
+};
+
+export type ChainsawPagingParams = {
+  pageSize?: number;
+  page?: number;
+};
+
+export type Block = {
+  hash: Hex;
+  blockNumber: number;
+  time: Date;
+  parentHash: Hex;
+  miner: Hex;
+  nonce: Hex;
+  baseFeePerGas?: string;
+  difficulty: string;
+  gasLimit: string;
+  gasUsed: string;
+  chainId: number;
+  version: number;
+};
+
+export type Event = {
```

Comment:
Do any of the event params come through?

Reply:
They didn't come before, but I made grouping optional now so if no grouping is specified, the `args` of each event will come through
[72fc01d](https://github.com/thirdweb-dev/js/pull/3741/commits/72fc01d1807bd106952b82297d8b7e2294a39bae)